### PR TITLE
Implement Jump +5  / -5 sec in Main Activity

### PR DESCRIPTION
**Pull request by AmrDeveloper**
Replace setAudioStreamType with setAudioAttributes because it deprecated

Implement Jump +5 / -5 Feature in MainActivity

#212

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -3,7 +3,7 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 
-def keystorePropertiesFile = rootProject.file("keystore.properties")
+def keystorePropertiesFile = rootProject.file("keystore.properties_sample")
```
Comment (tibbi) [inline app/build.gradle]: you definitely shouldnt break app signing with this one. Keep that change local.

Comment (AmrDeveloper) [reply app/build.gradle]: You are right i forgot to change it before pull request

Comment (AmrDeveloper) [reply app/build.gradle]: @tibbi Done i deleted this change

**Comment by tibbi** [comment]
okay, I will take a look at it a bit later, Im not working on this app right now. Anyway, you can already jump forward/backward by 2% by clicking on the current/max times.

**Comment by AmrDeveloper** [comment]
Ok thanks bro and nice organization

**Comment by tibbi** [comment]
I dont really like the way it floods the UI. I dont want to add such buttons now, you can already jump forward/backward by pressing the current/max duration. It works though, so Ill keep the code for reference. Thanks

**Pull request closed without merging**